cmd/talosctl/cmd/talos/edit.go

```diff
@@ -83,11 +88,7 @@ func editFn(c *client.Client) func(context.Context, string, resource.Resource, e
 			}
 
 			if lastError != "" {
-				lines := strings.Split(lastError, "\n")
-
-				_, err = w.Write([]byte(
-					fmt.Sprintf("# \n# %s\n", strings.Join(lines, "\n# ")),
```

Comment:
I think this had some `CRLF` magic for Windows due to the special writer being used which is not handled by the new function

Reply:
it's the same code as the old one

Reply:
https://github.com/siderolabs/talos/blob/e3c96117536a7dc55fb2f5817b1631819b7c6534/cmd/talosctl/cmd/talos/edit.go#L77 it's here

Reply:
ah... I see, sorry, we're still using the `w.Write`... I somehow looks at the signature and ignored that